src/medias/images/UGT_Asset_UI_Dropdown.tsx

```diff
@@ -7,9 +7,22 @@ export interface ImgDropdownProps extends React.SVGProps<SVGSVGElement> {
 
 export const ImgDropdown: React.FunctionComponent<ImgDropdownProps> = ({ alt, fill, ...props }) => {
   return (
-    <svg width="12" height="8" viewBox="0 0 12 8" fill="none" xmlns="http://www.w3.org/2000/svg" {...props}>
+    <svg
+      width="14"
+      height="8"
+      viewBox="0 0 14 8"
+      fill="none"
+      xmlns="http://www.w3.org/2000/svg"
+      {...props}
+    >
       {Boolean(alt) && <title>{alt}</title>}
-      <path d="M12 1.41L10.59 -6.16331e-08L6 4.58L1.41 -4.62904e-07L-6.16331e-08 1.41L6 7.41L12 1.41Z" fill={fill ?? "var(--color-secondary)"} />
+      <path
+        d="M13 1L7 7L1 1"
```

Comment:
You should try to use expanded strokes. This leaves less interpretation to the SVG implementation.
But might just be that Figma provided it this way?

Reply:
Yes, 🤔  Figma provided. (Figma SVG below)
<svg width="14" height="8" viewBox="0 0 14 8" fill="none" xmlns="http://www.w3.org/2000/svg">
<path d="M13 1L7 7L1 1" stroke="#0D1234" stroke-width="1.5" stroke-linecap="round" stroke-linejoin="round"/>
</svg>

and slightly altered as previous & other tsx files (example).
<img width="1194" alt="image" src="https://user-images.githubusercontent.com/96022735/161597762-afae4dfd-5872-4bf8-8dcc-b3202e1256ed.png">

🤔

Reply:
That's fine 😃 We can just leave it 👍